astroquery/gaia/core.py

```diff
@@ -853,15 +848,155 @@ def load_user(self, user_id, *, verbose=False):
 
         return self.is_valid_user(user_id=user_id, verbose=verbose)
 
+    def cross_match_basic(self, *, table_a_full_qualified_name, table_a_column_ra, table_a_column_dec,
+                          table_b_full_qualified_name=MAIN_GAIA_TABLE, table_b_column_ra=MAIN_GAIA_TABLE_RA,
+                          table_b_column_dec=MAIN_GAIA_TABLE_DEC, results_name=None,
+                          radius=1.0, background=False, verbose=False):
+        """Performs a positional cross-match between the specified tables.
+
+        This methods simples the execution of the method `cross_match` since it carries out the following steps in one
+        step:
+
+            #. updates the user table metadata to flag the positional RA/Dec columns;
+            #. launches a positional cross-match as an asynchronous query;
+            #. returns all the columns from both tables plus the angular distance (deg) for the cross-matched sources.
```

Comment:
Unfortunately this won't be auto resolved into numbers in the docstrings, only when sphinx rendered, so could you please instead spell out the numberings?